### ISSUE TITLE
Switch from `iro` color picker to `input type="color"`

### ISSUE THREAD

**Issue by with-heart** [Soil-labs/eden-app-2-FrontEnd#703]
The `ColorPicker` component is currently using https://github.com/jaames/iro.js to display the color wheel which allows users to pick a color.

Here it is in the create project view:

<img width="109" alt="image" src="https://user-images.githubusercontent.com/1954752/206620817-890ff62f-7582-4dcf-8521-5d0e9d4c65c8.png">

It's a neat idea but unfortunately it's completely inaccessible. It just renders three divs with a conic gradient of colors inside that translates clicks to a color. 

How would a person using a screenreader or a keyboard select a color? Well, they wouldn't—that's how.

The component is also not currently being maintained:

> NOTE: this project is currently on [hiatus](https://github.com/jaames/iro.js/issues/217#issuecomment-1214403290) until further notice, as working on open source projects is becoming unsustainable for the maintainer.

We should consider using the native [`input type="color"`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/color) element in its place. 

It's not only accessible (since it's native), it can also include additional features (depending on the browser):

- color palette
- color slider
- native system eyedropper tool
- rgb/hsl/hex inputs (so projects can match their brand color exactly)

Here's what it looks like in Arc on my Macbook:

<img width="235" alt="image" src="https://user-images.githubusercontent.com/1954752/206621654-cfd30533-b605-42dd-816b-2acf3af64b51.png">

It's triggered by a button with the current color in it but we can style this:

<img width="57" alt="image" src="https://user-images.githubusercontent.com/1954752/206621740-e34f51e8-347f-4079-9638-7468eef06046.png">

**Comment by with-heart** [comment]
btw I discovered this thanks to `testing-library` and its a11y-focused queries. Since there's no `getBy*` method for selecting what `iro` renders, it served as a big clue that it's not accessible.
